Makefile

```diff
@@ -184,34 +177,40 @@ test:
 test-windows:
 	go test -v $(go list ./... | grep -v /nodeup/)
 
+
+.PHONY: kops
+kops: crossbuild-$(shell go env GOOS)-$(shell go env GOARCH)
```

Comment:
Maybe we should be more explicit and use `crossbuild-kops-...`, or just drop `crossbuild-` completely?

Reply:
I thought about it, but didn't want to change the naming of targets in this PR. But it is something we should look at.

Reply:
I think the only one that already exists is `crossbuild`, all the rest are new.